README.md

```diff
@@ -18,7 +18,7 @@ chart types, including 3D charts, statistical graphs, and SVG maps.
 * [Documentation](#documentation)
 * [Contributing](#contributing)
 * [Community](#community)
-* [Clients for R, Python, and MATLAB](#clients-for-r-python-and-matlab)
+* [Clients for R, Python, and MATLAB](#clients-for-r-python-node-and-matlab)
```

Comment:
done already: https://github.com/plotly/plotly.js#clients-for-r-python-node-and-matlab